backend/schemas/output/sections/GeneralInformation.schema.json

```diff
@@ -131,6 +131,11 @@
       "audit_period_covered": {
          "$ref": "#/$defs/AuditPeriod"
       },
+      "audit_period_other_months": {
+         "max": 19,
```

Comment:
I had comments/questions here on the draft. Do we know why it is 18 months? That needs to be answered. Did @Carley-Sullivan or someone with more insight answer conclusively what the upper bound on this should be? I, at the moment, do not know.

And, could we use either `maximum` or `exlusiveMaximum` per the docs?

https://json-schema.org/understanding-json-schema/reference/numeric.html#range